drivers/sensor/wsen_hids_2525020210002/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+config WSEN_HIDS_2525020210002
+	bool "WSEN-HIDS-2525020210002 humidity sensor"
+	default y
+	depends on DT_HAS_WE_WSEN_HIDS_2525020210002_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_WE_WSEN_HIDS_2525020210002),i2c)
```

Comment:
Does this device support SPI or any other non-I2C interface? If not, you can remove the conditional if